payments_redsys/__init__.py

```diff
@@ -1,259 +1,292 @@
-# Copyright (C) 2018 AJ Ostergaard
-#
-# This file is part of django-payments-redsys.
-#
-# django-payments-redsys is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# django-payments-redsys is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with django-payments-redsys.  If not, see <https://www.gnu.org/licenses/>.
-
-from __future__ import unicode_literals
-import zeep

```

Comment:
I've migrated to use the rest endpoint, so no longer need the webservice dependencies.